### DISCUSSION TITLE
AGH and Unbound in the same Docker network

### DISCUSSION THREAD

**Discussion by agneevX** [AdguardTeam/AdGuardHome#3302] [Q&A]
If AdGuard Home and Unbound are in the same Docker network, how do I add Unbound to AGH?

None of these seem to work:

```
## Upstream servers
# Unbound listens on port 53

unbound
unbound:53
udp://unbound:53
```

**Comment by ameshkov** [comment]
What is this `unbound` in those upstream examples? Usually, you need to specify the IP address of the upstream.

**Reply by agneevX** [reply]
That's the hostname, since its DHCP by default.

**Reply by ameshkov** [reply]
AGH won't consult DHCP hostnames when connecting to upstream servers

**Reply by agneevX** [reply]
So it's best if both use `host` networking mode? Got it.
